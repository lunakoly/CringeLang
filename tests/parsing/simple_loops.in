while doThings * 10 + 1
    a = 10
    b * test

while a b = 10
    testError

done