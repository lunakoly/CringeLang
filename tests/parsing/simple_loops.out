==== Diagnostics ====
 5 | ...<newline>    testError<newline><newline>...
                 ~~~~^~~~~~~~~
Error > Line 5 > Unexpected indent level > `INDENT` shouldn't go here.

==== AST ====
[while ((doThings * 10) + 1) [[([a] = [10]), [(b * test)]]], while a [([b] = [10])], [[testError]], [done]]

==== RESOLVED ====
*** FILE 0 ***
[while ((doThings * 10) + 1) [[([a] = [10]), [(b * test)]]], while a [([b] = [10])], [[testError]], [done]]

==== Done ====
